### PR TITLE
[15.0][MIG] hr_contract_multi_job

### PR DESCRIPTION
**Pull request by yaseentai**

### CODE REVIEW

File: hr_contract_multi_job/models/hr_contract.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    _inherit = "hr.contract"
+
+    contract_job_ids = fields.One2many("hr.contract.job", "contract_id", string="Jobs")
+
+    # Modify the job_id field so that it points to the main job
+    job_id = fields.Many2one(
+        "hr.job", string="Job Title", compute="_compute_main_job_position", store=True
+    )
+
+    @api.depends("contract_job_ids")
```
Comment (hieulucky111) [inline hr_contract_multi_job/models/hr_contract.py]: must be depended on `contract_job_ids.is_main_job`

File: hr_contract_multi_job/models/hr_contract.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    def _compute_main_job_position(self):
+        """
+        Get the main job position from the field contract_job_ids which
+        contains one and only one record with field is_main_job == True
+        """
+        for contract in self:
+            main_job = contract.contract_job_ids.filtered("is_main_job").mapped(
+                "job_id"
+            )
+            if main_job and len(main_job) == 1:
```
Comment (hieulucky111) [inline hr_contract_multi_job/models/hr_contract.py:28]: must have `else` while compute function from v13 must always set the value.

File: hr_contract_multi_job/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_contract_job_manager,hr.contract.job.manager,model_hr_contract_job,hr.group_hr_manager,1,1,1,1
```
Comment (hieulucky111) [inline hr_contract_multi_job/security/ir.model.access.csv:2]: should have read access for normal user? `base.group_user`.

**Review by hieulucky111 — CHANGES_REQUESTED** [review]

**Review by Amjdhassan — APPROVED** [review]

**Review by Amjdhassan — APPROVED** [review]

**Review by hieulucky111 — APPROVED** [review]
Code review
LGTM

**Pull request merged**